packages/strapi-plugin-upload/admin/src/components/UploadForm/index.js

```diff
@@ -0,0 +1,68 @@
+import React, { useReducer } from 'react';
+import PropTypes from 'prop-types';
+import { HeaderNavWrapper, ModalBody, ModalForm } from 'strapi-helper-plugin';
+import ModalNav from '../ModalNav';
```

Comment:
Strange to have some of the modal components came from the strapi-helper-plugin and others from this directory.

Reply:
Well it is because the design differs from the usual ones. So I recreated new components thzat overrides the other ones.